fuse/requirements.txt

```diff
@@ -6,33 +6,28 @@ pandas>=1.2
 tqdm>=4.52.0
 scipy>=1.5.4
 matplotlib>=3.3.3
-scikit-image>=0.17.2
 scikit-learn>=0.23.2
 termcolor>=1.1.0
-torch>=1.5.0,<=1.11.0
+torch>=1.5.0,<=1.11.0 # the higher limit is temporary - to avoid from error in unitests:  "Cannot re-initialize CUDA in forked subprocess. To use CUDA with multiprocessing, you must use the 'spawn' start method"
```

Comment:
can you explain a bit more?
Not sure it's a good idea to limit the max pytorch version of the users...
By "temporary" do you mean it is intended to be resolved before fuse2 official release ?

Reply:
Not too much.
The last builds failed for us. All the unittest running in a child process failed with this error "Cannot re-initialize CUDA in forked subprocess. To use CUDA with multiprocessing, you must use the 'spawn' start method".
Probably something new trigger CUDA init on torch version 1.12.0. 
This is a workaround until we figure out what it is.

Reply:
What about the following suggestion:
Not limit the user pytorch version, and in our unit tests use an env var that run_multiprocessed reads and actually use the 'spawn' start method ? does that make sense ?

Reply:
I tried it first, naively changing it to "spawn" failed (trying to serialize something).
Agree that it shouldn't be our long term solution.
It's just to release fuse2 - and not to block all active pull requests.